tests/Behat/Tests/Config/ConfigTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Behat\Tests\Config;
+
+use Behat\Config\Config;
+use PHPUnit\Framework\TestCase;
+
+final class ConfigTest extends TestCase
+{
+    public function testConfigCanBeConvertedIntoAnArray(): void
+    {
+        $config = new Config();
+
+        $this->assertIsArray($config->toArray());
+    }
+
+    public function testItReturnsSettings(): void
+    {
+        $settings = [
```

Comment:
Not really a comment on this line, but I think we'd need at least some .feature coverage that the config is actually parsed and used as expected (not just asserting which file is picked).

This is where I imagine maintenance burden will start to creep in if the `Config` object adds more features, as we would need to be sure the `.php` and `.yaml` files produced the same effect.

Reply:
Yes, I've added a first scenario using a PHP file configuration.